src/components/ToggleGroup.jsx

```diff
@@ -0,0 +1,29 @@
+import React from 'react'
+import { ToggleButton, ToggleButtonGroup } from '@mui/material'
+import { useGuesses } from '../context/GuessContext' // see line 11
+
+
+export const ToggleGroup = ({ values, labels, handleChange }) => {
+
+    const { name, value1, value2, value3 } = values
+    const { groupLabel, buttonLabel1, buttonLabel2, buttonLabel3 } = labels
+
+    const { guesses } = useGuesses() // ? pass as prop from parent [also has useGuesses()] instead?
+
+    return (
+        <ToggleButtonGroup
+            fullWidth
+            value={guesses[name]}
+            exclusive
+            color="warning"
+            onChange={handleChange}
+            aria-label={groupLabel}
+        >
+            <ToggleButton name={name} value={value1} aria-label={buttonLabel1}>{value1}</ToggleButton>
```

Comment:
You have 3 buttons so I would use an array, then you can map over it. Pass the array of 'options' as props then use the index to pick out the data you need